src/components/recordingcreator/recordingfields.js

```diff
@@ -162,7 +164,7 @@ function onManageRecordingClick(e) {
     }
 
     const self = this;
-    import('recordingEditor').then(({default: recordingEditor}) => {
+    import('./recordingeditor').then((recordingEditor) => {
```

Comment:
```suggestion
    import('./recordingeditor').then(({default: recordingEditor}) => {
```
or we should simply export that `show` function from each `recording...` components.